pkg/api/convertertoagentpoolonlyapi.go

```diff
@@ -230,7 +230,7 @@ func convertV20170831AgentPoolOnlyOrchestratorProfile(kubernetesVersion string)
 			EnableRbac:          helpers.PointerToBool(false),
 			EnableSecureKubelet: helpers.PointerToBool(false),
 			// set network default for un-versioned model
-			NetworkPolicy:      string(v20180331.Kubenet),
+			NetworkPlugin:      string(v20180331.Kubenet),
```

Comment:
It seems to break backward compatibility on existing AKS which has `NetworkPolicy`